tests/integration/cattletest/core/common_fixtures.py

```diff
@@ -64,6 +64,48 @@ def super_client(request):
     return _get_super_client(request)
 
 
+@pytest.fixture(scope='module')
```

Comment:
What is all this? you are return the user_client for each one of these method, which are all exactly the same